dpsim-models/src/CIM/Reader.cpp

```diff
@@ -689,6 +691,14 @@ TopologicalPowerComp::Ptr Reader::mapSynchronousMachine(CIMPP::SynchronousMachin
 								Ld, Lq, Ll, Ld_t, Lq_t, Td0_t, Tq0_t,
 								Ld_s, Lq_s, Td0_s, Tq0_s);
 							return gen;
+						} else if (mGeneratorType == GeneratorType::SG5bOrderVBR) {
+							mSLog->info("    GeneratorType is SynchronGenerator5bOrderVBR.");
+							auto gen = std::make_shared<SP::Ph1::SynchronGenerator5bOrderVBR>(machine->mRID, machine->name, mComponentLogLevel);
+							gen->setOperationalParametersPerUnit(
+								ratedPower, ratedVoltage, mFrequency, H,
+								Ld, Lq, Ll, Ld_t, Lq_t, Td0_t, Tq0_t,
+								Ld_s, Lq_s, Td0_s, Tq0_s, 0.0); 
+							return gen;
```

Comment:
Can we add this to `EMT` and `DP` domain reading, since the models are also covered by this PR?